_studio/mfx_lib/shared/src/libmfxsw_plugin.cpp

```diff
@@ -201,6 +201,12 @@ mfxStatus MFXVideoUSER_Register(mfxSession session, mfxU32 type,
             return MFX_ERR_UNDEFINED_BEHAVIOR;
         }
 
+#ifndef MFX_ENABLE_USER_VPP
```

Comment:
I rearranged checks under MFX_ENABLE_USER_VPP and MFX_ENABLE_VPP to make more sense...

However, I completely don't understand why someone may wish to generate the library completely without VPP or user VPP plugin support?! Well, I understand that someone may wish to avoid some features, especially CM kernels, but we need to have separate ENABLE for that. Thus, I suggest to actually remove both of these ENABLES completely from the whole mediasdk.

@onabiull : how do you think? who we may need to ask about that? maybe @jonrecker?

Reply:
I believe this is artifact of attempts to create bare runtime library. In current state there is no need in these defines.